controllers/loadtest_controller.go

```diff
@@ -73,7 +72,29 @@ func (r *LoadTestReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 		return ctrl.Result{}, client.IgnoreNotFound(err)
 	}
 
+	loadTestTTL := time.Duration(rawTest.Spec.TTLSeconds) * time.Second
+	loadTestTimeout := time.Duration(rawTest.Spec.TimeoutSeconds) * time.Second
+
+	if loadTestTTL == 0 {
+		log.Info("TTL is invalid", "TTL", loadTestTTL)
+	}
+
+	if loadTestTimeout == 0 {
+		log.Info("Timeout is invalid", "Timeout", loadTestTimeout)
+	}
+
+	if loadTestTimeout > loadTestTTL {
+		log.Info("loadTestTTL is less than loadTestTimeout", "Timeout", loadTestTimeout, "TTL", loadTestTTL)
```

Comment:
nit: logs can be rendered as plain text or json. Let's make sure the fields, like the TTL all have the same case and name in all the log statements. I would suggest `ttl` or `testTTL`. JSON keys don't often start with caps.

Reply:
I will check!